bool app_enabled
string app_msg

string location_str

bool classifier_running

string image_topic

string[] available_classes_list
string[] selected_classes_list

float32 sensitivity 
bool snapshot_enabled
float32 snapshot_delay_sec 

bool active_alert
